eng/Directory.Build.Common.props

```diff
@@ -172,6 +180,7 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)AzSdkTestLibKey.snk</AssemblyOriginatorKeyFile>
     <!-- Disable doc comments for test projects -->
     <DocumentationFile></DocumentationFile>
+    <RunSettingsFilePath>$(RepoEngPath)\nunit.runsettings</RunSettingsFilePath>
```

Comment:
@JoshLove-msft as part of merging your PR https://github.com/Azure/azure-sdk-for-net/pull/19842 with mine I moved your new setting here. Are you aware of any reason this wouldn't work?

Reply:
As long as this would get pulled into all the sdk directories it should work.

Reply:
Yes it should.